shared/message/platform/NUSenseData.proto

```diff
@@ -62,6 +62,14 @@ message Servo {
     float voltage = 12;
     /// The last read temperature of the servo
     float temperature = 13;
+    /// The number of successes.
+    uint32 num_successes = 14;
```

Comment:
I don't see this used anywhere in these changes. Does this get used in the nusense firmware? Also, do we specifically need a proto field for the number of successes? This can still be tracked by saying that a packet that did not return any error means that it succeeded sending the commands to the servos and it can just be tracked by a uint32 variable in the header file of hwio.

Reply:
Sorry, I should have linked this which is the sister PR. This won't be merged until the other PR is approved as well. https://github.com/NUbots/NUcontroller/pull/16

The number of successes is just in case we want to infer the total number of packets, i.e. `num_success + num_crc_errors + num_errors`, in order to maybe use a percentage, e.g. "5% of packets received had a CRC error". 

Also, keep in mind that HardwareIO does not know anything about individual dynamixel packets. For every `NUSenseData` packet sent along USB, there could be at least a dozen dynamixel packets for each servo. So, the only way to infer packet-statistics is to log them on NUSense and send them over.

Reply:
Another thing for that it could be used is to infer and to monitor the dynamixel throughput.

Reply:
Do you want to count the number of successes? or the total number of packets?
Also I'd make these comments for this and errors that it is just the number of errors since the last packet, so it doesn't grow over time.

Reply:
Will change to total number instead.